src/docs/command-line.dox

```diff
@@ -341,6 +342,9 @@ output; the \c -f option re-directs the output to the specified file.
 @par <code>-m</code>
 Print only message-type log records.
 
+@par <code>-u</code>
+Print user data. Do not redact.
```

Comment:
This feels like a weird pair of sentence fragments, maybe just "display user data"?

Reply:
Fixed.